### PR TITLE
Remove codalabworker package

### LINKED ISSUE #1346: Get rid of codalabworker package and just have one codalab package
This will simplify the currently convoluted dependencies between codalab and codalabworker.

### PR DESCRIPTION
**Pull request by candicegjing**
Fixed #1346 .
I re-created this PR as the previous one was end up in *partial-merge-pull* state.

### CODE REVIEW

File: docker/dockerfiles/Dockerfile.worker [hunk 1]
```diff
@@ -6,22 +6,26 @@ ENV DEBIAN_FRONTEND noninteractive
...
-# Install dependencies again
-RUN python3.6 -m pip install --user --upgrade pip; \
-    python3.6 -m pip install -e /opt/worker;
+RUN python3.6 -m pip install -e .
 
 # Allow non-root to read everything
-RUN chmod -R og=u-w /opt/worker
+RUN chmod -R og=u-w .
+
+EXPOSE 2900
```
Comment (percyliang) [inline docker/dockerfiles/Dockerfile.worker]: Why is this here?  this is from the rest server

**Review by percyliang — APPROVED** [review]

File: codalab/worker/local_run/local_dependency_manager.py [hunk 2]
```diff
@@ -186,7 +186,7 @@ def _cleanup(self):
             with self._global_lock:
                 self._acquire_all_locks()
                 bytes_used = sum(dep.size_bytes for dep in self._dependencies.values())
-                serialized_length = len(codalabworker.pyjson.dumps(self._dependencies))
+                serialized_length = len(codalab.pyjson.dumps(self._dependencies))
```
Comment (epicfaace) [inline codalab/worker/local_run/local_dependency_manager.py]: Back to my comment on the original PR: why do we have codalab.pyjson instead of codalabworker.pyjson?

Comment (candicegjing) [reply codalab/worker/local_run/local_dependency_manager.py]: This is different than the original PR. I merged something wrong in this PR. It should be `codalab.worker`

File: docker/dockerfiles/Dockerfile.worker (hunk 1 above)
Comment (candicegjing) [reply docker/dockerfiles/Dockerfile.worker]: It shouldn't be here. I was testing something else and left it here. I removed it just now.
